app/Controllers/Http/ProfilesController.ts

```diff
@@ -0,0 +1,70 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Database from '@ioc:Adonis/Lucid/Database'
+import Profile from 'App/Models/Profile'
+import UpdateProfileValidator from 'App/validators/UpdateProfileValidator'
+
+export default class ProfilesController {
+  public async getProfile({ auth, response }: HttpContextContract) {
+    try {
+      const email = auth.user?.email
+      const user = await Database.from('user as u')
+        .innerJoin('profile as p', 'u.id', 'p.user_id')
+        .where({ email })
+        .select(['email', 'mobile', 'gender', 'dob', 'name'])
+        .first()
```

Comment:
instead of this use ORM. 

`auth.user?.load('profile')`

Reply:
fixed